test/runtests.jl

```diff
@@ -13,15 +14,24 @@ PyCall.pyimport_conda("xlrd", "xlrd")
 
 py"""
 import os
+import pandas as pd
```

Comment:
Remove `import pandas` and put it in a file(regarding).

Reply:
Can you elaborate more on this?